src/Mutant/DetectionStatus.php

```diff
@@ -54,6 +54,9 @@ final class DetectionStatus
     public const NOT_COVERED = 'not covered';
     public const IGNORED = 'ignored';
 
+    /**
+     * @var array<self::*>
+     */
```

Comment:
is it needed? `make phpstan` works without it, while `make psalm` fails with this

Reply:
We can drop it I guess